test/LibraryManager.Test/Providers/JsDelivr/JsDelivrCatalogTest.cs

```diff
@@ -291,6 +299,46 @@ public async Task GetLatestVersion_PreRelease()
 
             Assert.AreEqual("2.0.0-beta", result);
         }
+
+        [TestMethod]
+        public async Task GetLibraryCompletionSetAsync_ReturnsCompletionWithLatestVersion()
+        {
+            //Arrange
+            var dependencies = new Dependencies(null);
+            var provider = new Provider(null) { Id = JsDelivrProvider.IdText, SupportsLibraryVersions = true };
+
+            dependencies.AllProviders = new List<IProvider>() { provider };
+
+            //Reinitialize is necessary because any other tests could have already initialized before
+            LibraryIdToNameAndVersionConverter.Instance.Reinitialize(dependencies);
```

Comment:
This becomes unnecessary by using the instance field above.  (That was part of #498, to avoid this level of state-dependent setup.)

Reply:
Resolved in commit 4